rgb_camera:
  size_img_x: 640
  size_img_y: 480
  x0: 319.5
  y0: 239.5
  fx: 554.254691191187
  fy: 554.254691191187

depth_camera:
  size_img_x: 640
  size_img_y: 480
  x0: 319.5
  y0: 239.5
  fx: 554.254691191187
  fy: 554.254691191187
